models/desired_lrp_test.go

```diff
@@ -141,7 +143,8 @@ var _ = Describe("DesiredLRP", func() {
             },
             "user": "vcap",
             "log_source": "HEALTH",
-			"suppress_log_output": true
+			"suppress_log_output": true,
+		    "volume_mounted_files": null
```

Comment:
I see that this tabbing was weird before you were in the file, but can you fix it since you are in this file?